src/components/PostDetails.tsx

```diff
@@ -1,106 +1,121 @@
-import React from 'react';
-import { Loader } from './Loader';
+import React, { useEffect } from 'react';
 import { NewCommentForm } from './NewCommentForm';
+import { Loader } from './Loader';
+import { Comment, CommentData } from '../types/Comment';
+import { User } from '../types/User';
+import { Post } from '../types/Post';
+
+type Props = {
+  isCommentLoading: boolean;
```

Comment:
The naming of the prop `isCommentLoading` is not clear. It can be `isLoadingComments` or `isCommentsLoaded` to make it more readable and understandable.